Source/Classes/PINRemoteImageDownloadQueue.h

```diff
@@ -0,0 +1,37 @@
+//
+//  PINRemoteImageDownloadQueue.h
+//  PINRemoteImage
+//
+//  Created by Garrett Moon on 3/1/17.
+//  Copyright © 2017 Pinterest. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+#import "PINRemoteImageManager.h"
+
+@class PINURLSessionManager;
+
+NS_ASSUME_NONNULL_BEGIN
+
+typedef void (^PINRemoteImageDownloadCompletion)(NSURLResponse *response, NSError *error);
+
+@interface PINRemoteImageDownloadQueue : NSObject
+
+@property (nonatomic, assign) NSUInteger maximumNumberOfOperations;
+
+- (instancetype)init NS_UNAVAILABLE;
+- (PINRemoteImageDownloadQueue *)initWithMaximumNumberOfOperations:(NSUInteger)maximumNumberOfOperations NS_DESIGNATED_INITIALIZER;
```

Comment:
Consider a +queueWithMax:... (but only if this suits the framework's API design patterns).  It would make the creation line fit a bit more elegantly with a long name.